jar-service/src/main/java/com/sixsq/slipstream/application/RootApplication.java

```diff
@@ -351,39 +349,24 @@ private void attachAction(RootRouter router) {
 	}
```

Comment:
Yes, because we are using it directly into `router.attach(...)` instead of first passing it to `convertToRouterRoot(...)`.